adam-python/setup.py

```diff
@@ -16,15 +16,119 @@
 # limitations under the License.
 #
 
+from __future__ import print_function
+import glob
+import os
+import sys
 from setuptools import find_packages, setup
+
 from version import version as adam_version
 
-setup(
-    name='bdgenomics.adam',
-    version=adam_version,
-    description='A fast, scalable genome analysis system',
-    author='Frank Austin Nothaft',
-    author_email='fnothaft@berkeley.edu',
-    url="https://github.com/bdgenomics/adam",
-    install_requires=[],
-    packages=find_packages(exclude=['*.test.*']))
+if sys.version_info < (2, 7):
+    print("Python versions prior to 2.7 are not supported for pip installed ADAM.",
+          file=sys.stderr)
+    exit(-1)
+
+# Provide guidance about how to use setup.py
+incorrect_invocation_message = """
+If you are installing PyADAM from ADAM's source, you must first build ADAM and
```

Comment:
We don't actually call this PyADAM anywhere, right?  Rather bdgenomics.adam, or Python support for ADAM, or somesuch.